pom.xml

```diff
@@ -180,19 +180,27 @@ THE SOFTWARE.
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>matrix-auth</artifactId>
+      <version>1.7</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency> <!-- TODO pending matrix-auth 2.x, thus Jenkins 2.60.x -->
+      <groupId>org.jenkins-ci.plugins.icon-shim</groupId>
+      <artifactId>icon-shim</artifactId>
+      <version>2.0.3</version>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
 
   <build>
     <plugins>
       <plugin>
         <groupId>org.jenkins-ci.tools</groupId>
         <artifactId>maven-hpi-plugin</artifactId>
```

Comment:
Is it required anymore? Not sure we need to mask classes

Reply:
No idea, but I did not see any reason to touch it in this PR.

Reply:
Well, you touched it. See the lines below

Reply:
I touched some plugin configuration as part of routine POM cleanup. Changing class loading behavior for one of this plugin’s libraries is a much riskier change, and totally unrelated to the rest of this PR.